rollingpin/args.py

```diff
@@ -211,15 +231,18 @@ def parse_args(config, raw_args=None, profile=None):
 
     args = parser.parse_args(args=raw_args)
 
+    args.commands = _parse_command_args(args.commands)
+
     if not args.restart:
         default_restart = config["deploy"].get("default-restart", [])
         if not isinstance(default_restart, list):
             default_restart = [default_restart]
 
-        args.restart = default_restart
-
-    for target in args.restart:
-        args.commands.append(["restart", target])
+        for target in default_restart:
```

Comment:
Not a big fan of this conditional but I couldn't really come up with a more clever way to do this. Any alternative would involve wiring the arguments through as tuples of some sort with the explicitness boolean set for the ones from argparse, which would be annoying and which honestly seems like its less obvious than just doing it this way.